src/ecalc/libraries/libecalc/common/libecalc/dto/result/tabular_time_series.py

```diff
@@ -51,6 +51,15 @@ def to_dataframe(
         return df
 
     def resample(self, freq: Frequency) -> Self:
+        """
+        Immutable - returns a copy of itself
+
+        Resample the given time series to the new Frequency given. Only data
+        that is defined as a timeseries will be resampled.
+
+        :param freq:
+        :return: return a copy of itself with all data resampled to given frequency
```

Comment:
Did we decide on using google doc-string format?

Pycharm does it automatically for me, I can show you if you would like

Reply:
Didnt make any changes since we discussed this, so forgot to change setup in pycharm.

Does interrogate and/or our scripts handle both or make sure we are consistent somehow?